ql/src/java/org/apache/hadoop/hive/ql/parse/type/FunctionHelper.java

```diff
@@ -78,6 +79,12 @@ AggregateInfo getWindowAggregateFunctionInfo(boolean isDistinct, boolean isAllCo
       String aggregateName, List<RexNode> aggregateParameters)
       throws SemanticException;
 
+  /**
+   * Returns RexCall for UDTF based on given parameters
+   */
+  RexCall getUDTFFunction(String functionName, List<RexNode> operands)
```

Comment:
nit.: IIUC this method is more than just getting an entry from the `FunctionRegistry` it also converts it to a `RexCall`